### ISSUE TITLE
Makefile problems

### ISSUE THREAD

**Issue by nlhowell** [climoy/field_morph#6]
```
$ make
make: *** No rule to make target 'numerals_isolated.gen.hfst', needed by 'check-gen'.  Stop.
```

**Comment by nlhowell** [comment]
Can you please update the makefile, replacing
"numerals_isolated.gen.hfst" with the correct transducer name?

**Comment by climoy** [comment]
374525053845c278950b511d0e7953c5619a1011 but i cannot figure out where is the error

**Comment by climoy** [comment]
oh I see, you've placed the resolution in 613953743a8c3fd15c351635c875e927a5533b30

**Comment by nlhowell** [comment]
Yeah, the problem there is syntax errors in twol, not in the makefile.
But if you manage to merge (`git pull
https://gitlab.com/nlhowell/agx.git master` should be enough?), it
should be fixed.

**Comment by climoy** [comment]
![image](https://user-images.githubusercontent.com/52608838/108719551-c2157c00-7530-11eb-876d-28963b531ae5.png)
I merged it but Make fails with this one:

**Comment by climoy** [comment]
this is my log, i tend to think that i've mirged in a right way but maybe something is wrong still 
![image](https://user-images.githubusercontent.com/52608838/108719874-289a9a00-7531-11eb-9007-84ee12edfa89.png)

**Comment by nlhowell** [comment]
roman - notifications@github.com wrote:
> ![image](https://user-images.githubusercontent.com/52608838/108719551-
> !c2157c00-7530-11eb-876d-28963b531ae5.png)
> I merged it but Make fails with this one:

I guess that `sh` is not a symlink to `bash`. Add to the top of your
makefile:
```
SHELL = /bin/bash
```

and it should work.

**Comment by nlhowell** [comment]
roman - notifications@github.com wrote:
> this is my log, i tend to think that i've mirged in a right way but
> maybe something is wrong still
> ![image](https://user-images.githubusercontent.com/52608838/108719874-
> !289a9a00-7531-11eb-9007-84ee12edfa89.png)

The merge looks good to me; see the comment about `SHELL`.

**Comment by climoy** [comment]
yes, it's woked out, thank you!
pushing the line in the front of Make 88e4b1d8348fb6c104bf5a9a2a0b5fa6333f8b5d
also edit .gitignore b7f0f10a9564a73f796d77fe2b2ae4b08ca0a562 (please kindly verify whether it's ok)
also test (single one):  65a86ceecf5f2264875dba9a78edbe7fa6aa69f4 im a bit shy writting them for now since need the validation of yours as well

**Comment by climoy** [comment]
also! need an assist in this way: after Make script has run, many of intermediary files present (see attach) resulting errors in future runs of Make. Is there any way of removing them automatically?
![image](https://user-images.githubusercontent.com/52608838/108725764-c4c79f80-7537-11eb-91fe-cb842676152e.png)

**Comment by nlhowell** [comment]
roman - notifications@github.com wrote:
> also, need an assist in this way: after Make script has run, many of
> intermediary files present (see attach) resulting errors in future runs
> of Make. Is there any way of removing them automatically?
> ![image](https://user-images.githubusercontent.com/52608838/108725764-
> !c4c79f80-7537-11eb-91fe-cb842676152e.png)

I don't particularly feel they need to be removed, but if you are
upset with them, `git clean -fx` will delete any untracked files.

And these files aren't interfering with future runs of make, I don't
think...
